src/main/resources/templates/mdBenefitsFlow/choosePrograms.html

```diff
@@ -17,15 +16,26 @@
             <div class="form-card__content">
               <th:block th:replace="~{fragments/inputs/checkboxFieldset ::
                   checkboxFieldset(inputName='programs',
-                  label=#{general.check-all-that-apply},
+                  label=#{choose-programs.check-all-that-apply},
                   content=~{::content})}">
                 <th:block th:ref="content">
-                  <th:block
-                      th:replace="~{fragments/inputs/checkboxInSet :: checkboxInSet(inputName='programs',value='SNAP', label=#{choose-programs.snap}, checkboxHelpText=#{choose-programs.snap-help})}"/>
+                  <th:block th:each="programType : ${T(org.mdbenefits.app.data.enums.ProgramType).values()}">
+                    <th:block th:if="${fieldData.get('helpNeeded[]').contains(programType.name())}">
+                      <span th:with="shortDesc=${programType.getShortDescSrc().isBlank()} ? '' : #{${programType.getShortDescSrc()}}"
```

Comment:
The one place where this PR deviates from the design is the styling of the help text on the Choose Program page.  The color is wrong and I could not easily figure out how to change it.